yarn.lock

```diff
@@ -12957,9 +12957,9 @@ react-native-scrollable-tab-view@^1.0.0:
     prop-types "^15.6.0"
     react-timer-mixin "^0.13.3"
 
-react-native-simple-crypto@RocketChat/react-native-simple-crypto:
+react-native-simple-crypto@RocketChat/react-native-simple-crypto#new.push-encryption:
```

Comment:
🚨 Don't merge before send the changes to `react-native-simple-crypto` master branch 🚨

Reply:
@djorkaeffalexandre Don't forget about this